### PR TITLE
DEVOPS-14092: add alb support

### PR DESCRIPTION
**Pull request by altuner**
Added tracking for TargetGroup, so now license2deploy can work with ALB

### CODE REVIEW

File: License2Deploy/rolling_deploy.py [hunk 1]
```diff
@@ -366,8 +410,10 @@ def stop_deploy(self, message='an error has occurred', e=None, error_code=2):
     exit(error_code)
 
   def deploy(self): # pragma: no cover
+
```
Comment (dnb-markusd) [inline License2Deploy/rolling_deploy.py:413]: Remove whitespace.

**Review by dnb-markusd — CHANGES_REQUESTED** [review]

File: License2Deploy/rolling_deploy.py (hunk 1 above)
Comment (altuner) [reply License2Deploy/rolling_deploy.py]: @dnb-markusd addressed

**Comment by dnb-markusd** [comment]
@altuner for testing, what stacks has this been tested with? Can you confirm / provide evidence that this is backwards compatible with the behavior expected by existing stacks?

**Comment by RudenkoM** [comment]
can you please add dome description to the PR

**Comment by altuner** [comment]
@RudenkoM done

File: License2Deploy/rolling_deploy.py [hunk 2]
```diff
@@ -358,6 +393,13 @@ def enable_project_cloudwatch_alarms(self):
         logging.error("Unable to enable the cloud-watch alarm, please investigate: {0}".format(e))
         exit(self.exit_error_code)
 
+  def get_target_group(self, asg_group):
+    target_groups = self.asg.describe_auto_scaling_groups(AutoScalingGroupNames=[asg_group])['AutoScalingGroups'][0]['TargetGroupARNs']
```
Comment (RudenkoM) [inline License2Deploy/rolling_deploy.py:397]: is there always one autoscaling group and one target group ?

File: License2Deploy/rolling_deploy.py [hunk 3]
```diff
@@ -358,6 +393,13 @@ def enable_project_cloudwatch_alarms(self):
...
+    if target_groups:
```
Comment (RudenkoM) [inline License2Deploy/rolling_deploy.py:398]: can this be collapsed into the ternary operator?

File: License2Deploy/rolling_deploy.py (hunk 2 above)
Comment (altuner) [reply License2Deploy/rolling_deploy.py:397]: could be more than one, but we're picking 1st one always

**Review by RudenkoM — APPROVED** [review]

File: License2Deploy/rolling_deploy.py [hunk 4]
```diff
@@ -57,6 +59,7 @@ def __init__(self,
     self.only_new_wait = only_new_wait
     self.asg_logical_name = asg_logical_name
     self.load_balancer = load_balancer
+    self.target_group_arn = False
```
Comment (mayn) [inline License2Deploy/rolling_deploy.py]: please use `None`

**Review by mayn — APPROVED** [review]

**Pull request merged**